experience.tex

```diff
@@ -6,33 +6,35 @@ \section{EXPERIENCE}
 \end{tabular*}
 \begin{bulletlist}
     \item{
-        Worked with the Supply Chain team to determine requirements and metric thresholds for displaying data to
-        the service providers based on their performance for previously completed work items.
+        Assisted building and maintaing greenfield iOS application written in SwiftUI to replace legacy Xamarin application,
+        reducing crash rate from 9\% to < 1\%, reducing time for local development and time to ship a new version of the app,
+        and added thorough logging to the application to make errors actionable for development team.
     }
     \item{
-        Utilized AWS Elasticsearch to improve filtering for assets by properties and to quickly return large lists of
-        matching data to the primary asset web service, lowering asset search response times for the end user.
+        Containerized legacy C\# applications to run in serverless AWS Fargate, moving secrets to HashiCorp Vault to be fetched
+        on container startup, and handled routing traffic securely from Route53, to ALB, to the ECS cluster for a modern
+        deployment and scalability.
     }
     \item{
-        Assisted in translating UI mockups created by the development, sourcing, and UX teams to their final frontend versions using randomly generated placeholder data.
+        Triaged bug reports from users, frequently shipping a new mobile application within an hour of the error being reported.
     }
 \end{bulletlist}
 
 \begin{tabular*}{\textwidth}{l@{\extracolsep{\fill}}r}
-    Software Engineer Intern & September 2018 – May 2019\\
-    Wickidcool Software & Milwaukee, WI\\
+    Consultant & July 2021 – October 2021\\
+    Rebentify Technologies & Milwaukee, WI\\
 \end{tabular*}
 \begin{bulletlist}
     \item{
-        Remotely collaborated with other developers using Git to minimize merge conflicts and implement features
-        simultaneously, improving productivity and reduced development timeline.
+        Set up staging and production environments in GCP Cloud Run and firebase to allow for seamless deployments through integrated
+        CI/CD pipelines, ensuring builds were shared across tiers with only configuration changes for security and reproducible behavior.
     }
     \item{
-        Collaborated in development for the front-end using ReactJS and Redux to handle data received from the
-        Node.js back-end of the application, populating it with data from other event services.
+        Integrated Login with Google into the website, allowing users to quickly login with their organization email for increased security
```

Comment:
Maybe say what it replaced and why Login with Google was an advantage - did you not have to maintain some custom integration with Google, or not retain student'a passwords?

Reply:
Resolved in a call with you